README.md

```diff
@@ -21,3 +21,15 @@ $ make
 
 [nasm]: https://www.nasm.us/
 [qemu]: https://www.qemu.org/
+
+## Building on MacOS
+
+1. `brew install nasm`
+2. `brew install qemu`
```

Comment:
AFAIK, `brew` can install multiple packages at once. Let's squash this into a single line like `brew install nasm qemu`, so it's easy to copy-paste.